src/components/MaxFeeAndSubmitToken/MaxFeeAndSubmitToken.vue

```diff
@@ -0,0 +1,114 @@
+<template>
```

Comment:
Why did you create a copy of the MaxFreeAndSubmit.vue component? Do you have any differences?

Reply:
to separate the button "Send" and give it different styles